app.yaml

```diff
@@ -24,6 +24,7 @@ handlers:
   expiration: 0s
   secure: always
   http_headers:
+    Content-Security-Policy: "base-uri 'self'; object-src 'none'; script-src 'self' https://fast.appcues.com https://us.jsagent.tcell.insight.rapid7.com https://cdnjs.cloudflare.com; style-src * 'unsafe-inline'"
```

Comment:
Do we need to include any of the other services in here, like TDR or Rawls?

Reply:
Only if JS or CSS is loaded from them.